Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -272,6 +272,91 @@ public class MongoDatabase {
         return try self._client.executeOperation(operation, session: session)
     }
 
+    /**
+     * Starts a `ChangeStream` on a database. Excludes system collections. By default, the type `CollectionType` is
+     * associated with the `fullDocument` field in `ChangeStreamsDocument`.
+     * - Parameters:
+     *   - Pipeline: The pipeline of stages to append to an initial `ChangeStream` stage.
+     *   - Options: An optional `ChangeStreamOptions` to use on the initial `ChangeStream` stage.
+     *   - Session: An optional `ChangeStream` to use with this change stream.
+     * - Throws:
+     *   - `ServerError.commandError` if an error occurs on the server while creating the cursor.
+     *   - `UserError.invalidArgumentError` if the options passed formed an invalid combination or the pipeline passed
+     *     is invalid.
+     *   - `UserError.invalidArgumentError` if the `_id` field is projected out of the change stream documents by the
+     *     pipeline.
+     */
+     public func watch(_ pipeline: [Document],
+                       options: ChangeStreamOptions? = nil,
+                       session: ClientSession? = nil) throws ->
+                       ChangeStream<ChangeStreamDocument<Document>> {
+        return try self.watch(pipeline, options: options, session: session, withFullDocumentType: Document.self)
+     }
+
+    /**
+     * Starts a `ChangeStream` on a database. Excludes system collections. Associates the specified `Codable` type `T`
+     * with the `fullDocument` field in the `ChangeStreamDocument`.
```

Comment:
I'd amend this docstring to say "...`ChangeStreamDocument`s emitted by the returned `ChangeStream`." This also applies to other docstrings for similar watch methods on client/collection.

Also the docstring alignment is off by one. The top '/' should be aligned with the 'p' in `public`.